plugin/storage/grpc/factory_test.go

```diff
@@ -120,7 +121,11 @@ func TestNewFactoryError(t *testing.T) {
 
 	t.Run("client", func(t *testing.T) {
 		// this is a silly test to verify handling of error from grpc.NewClient, which cannot be induced via params.
```

Comment:
should we just ~delete~ comment-out this test? (with TODO and an issue to restore later once grpc fix is done).

Reply:
@yurishkuro I think its okay to leave it because `DialContext` calls `NewClient`